test/e2e_tests/test_loitering.py

```diff
@@ -0,0 +1,70 @@
+from pathlib import Path
+
+import vqpy
+from vqpy.class_names.coco import COCO_CLASSES
+from vqpy.operator.detector import register
+
+from e2e import FakeYOLOX, compare
+
+root = Path(__file__).parent.parent.parent
+video_name = "loitering"
+video_ext = "mp4"
+task_name = "loitering"
+fake_detector_name = f"fake-yolox-{video_name}"
+video_path = (root / f"videos/{video_name}.{video_ext}").as_posix()
+precomputed_path = (root / f"precomputed/{video_name}_yolox.pkl").as_posix()
+save_folder = (root / "e2e_outputs/").as_posix()
+result_path = (
+    root / f"e2e_outputs/{video_name}_{task_name}_{fake_detector_name}.json"
+).as_posix()
```

Comment:
Shall we generate a temporary file to save the result each time we run the test? 

The concern is, since you don't delete the result file of previous tests, you may end up testing the old result generated by previous tests, in the cases like the internal save file name changes, .etc